.eslintrc.js

```diff
@@ -205,9 +205,7 @@ module.exports = {
       },
     },
     {
-      files: [
-        "packages/{block-scripts,block-template,blockprotocol,create-block-app}/**",
-      ],
+      files: ["packages/**"],
```

Comment:
TODO: Replace with `"unicorn/filename-case": "error"` (global rule) when https://github.com/blockprotocol/blockprotocol/pull/304 is merged. I also need to remove this redundant rule: https://github.com/blockprotocol/blockprotocol/pull/304/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R323-R326